lessons/6_objects_intro/index.html

```diff
@@ -0,0 +1,332 @@
+    </style>
+</head>
+
+<body>
+<a href="https://www.redi-school.org/"><img src="../../redi-school-logo.jpg" class="logo"/></a>
+<div class="reveal">
+    <div class="slides">
+        <section>
+            <h1>Lecture</h1>
+            <h3>Intro to objects</h3>
+        </section>
+
+        <section>
+            <h3>Let's say you want to sell books on your website...</h3>
+            <p class="fragment">You start with 1 book</p>
+            <div class="fragment">
+                <div class="book">
+                    <img src="zombie_raccoons.jpg">
+                    <div>
+                        <div>Name: <span>Zombie Raccoons & Killer Bunnies</span></div>
+                        <div>Price: <span class="price">20 &euro;</span></div>
+                        <div>Length: <span class="pages">643 pages</span></div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section>
+            <h3>How your HTML might look like?</h3>
+            <div class="fragment">
+            <pre>
+					<code data-trim>
+<div class="book">
+	<div>
+    	<img class="image" src="zombie_raccoons.jpg">
+	</div>
+    <div>
+		<div>Name: <span>Zombie Raccoons & Killer Bunnies</span></div>
+		<div>Price: <span class="price">20 &euro;</span></div>
+		<div>Length: <span class="pages">643 pages</span></div>
+	</div>
+</div>
+                	</code>
+				</pre>
+            </div>
+        </section>
+
+        <section class="">
+            <h3>But all this information can come from elsewhere</h3>
+            <p class="fragment">Lest not hard-code it in HTML</p>
+            <div class="fragment">
+				<pre><code>
+let name = "Zombie Raccoons & Killer Bunnies";
+let image = "zombie_raccoons.jpg";
+let pageCount = 643;
+let price = 20;
+
+document.querySelector(".book img").src = image;
+document.querySelector(".book.name").innerText = name;
+document.querySelector(".book.price").innerText = price;
+document.querySelector(".book.pages").innerText = pageCount;
+          </code></pre>
+            </div>
+            <div class="fragment">
+            </div>
+
+        </section>
+
+        <section>
+            <h3>Time to add a second book!</h3>
+            <div class="fragment">
+                <div class="book-sample-img">
+                    <img src="zombie_raccoons.jpg">
+                    <img src="goats.jpg">
+                </div>
+                <br>
+                <div class="fragment">
+                    <pre><code data-trim>
+            let title1 = "Zombie Raccoons & Killer Bunnies";
+            let image1 = "zombie_raccoons.jpg";
+            let pageCount1 = 643;
+            let price1 = 20;
+
+            let title2 = "Raising goats for dummies";
+            let image2 = "goats.jpg";
+            let pageCount2 = 577;
+            let price2 = 15;
+          </code></pre>
+
+                </div>
+            </div>
+        </section>
+
+        <section class="">
```

Comment:
I think it would be nice to make situation even harder with "But in two weeks two new books arrived. What we gonna do now?" and then go to Objects to rescue.
What do you think?

Reply:
updated